upstream-community-operators/eunomia/0.1.0/eunomia.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1429 @@
+            selector:
+              matchLabels:
+                name: eunomia-operator
+            strategy: {}
+            template:
+              metadata:
+                labels:
+                  name: eunomia-operator
+              spec:
+                containers:
+                  - env:
+                      - name: WATCH_NAMESPACE
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: 'metadata.annotations[''olm.targetNamespaces'']'
+                      - name: POD_NAME
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.name
+                      - name: OPERATOR_NAME
+                        value: eunomia-operator
+                    image: 'quay.io/kohlstechnology/eunomia-operator:v0.1.0'
+                    imagePullPolicy: Always
+                    name: eunomia-operator
+                    resources:
+                      limits:
+                        cpu: 200m
+                        memory: 256Mi
+                      requests:
+                        cpu: 200m
+                        memory: 256Mi
+                serviceAccountName: eunomia-operator
+      permissions:
+        - rules:
+            - apiGroups:
+                - ''
+              resources:
+                - configmaps
+                - pods
+              verbs:
+                - '*'
+            - apiGroups:
+                - ''
+              resources:
+                - services
+              verbs:
+                - '*'
+            - apiGroups:
+                - apps
+              resources:
+                - replicasets
+                - deployments
+              verbs:
+                - '*'
+            - apiGroups:
+                - monitoring.coreos.com
+              resources:
+                - servicemonitors
+              verbs:
+                - get
+                - create
+            - apiGroups:
+                - apps
+              resourceNames:
+                - eunomia-operator
+              resources:
+                - deployments/finalizers
+              verbs:
+                - update
+          serviceAccountName: eunomia-operator
+    strategy: deployment
+  installModes:
+    - supported: true
+      type: OwnNamespace
+    - supported: true
+      type: SingleNamespace
+    - supported: false
+      type: MultiNamespace
+    - supported: true
+      type: AllNamespaces
+  keywords:
+    - gitops
+    - git
+    - management
+    - configuration
+  links:
+    - name: repository
+      url: 'https://github.com/KohlsTechnology/eunomia'
+    - name: conatinerImage
+      url: 'https://quay.io/kohlstechnology/eunomia-operator'
+  maintainers:
+    - email: heiko.zuerker@kohls.com
+      name: Heiko Zuerker
+    - email: sean.malloy@kohls.com
+      name: Sean Malloy
+  maturity: alpha
+  provider:
+    name: Kohls Technology
+  replaces: ''
```

Comment:
, please remove "replaces" as it is the first commit. Every newer version should replace latest one.

Reply:
Thanks for the feedback I have removed the replace field and replace the smaller image :)